xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_es.properties

```diff
@@ -2765,14 +2765,14 @@ xe.panels.rights.welcome=Bienvenido al editor de privilegios.
 xe.panels.rights.explanation=Los privilegios aplicados a un documento reemplazan los derechos aplicados a un espacio y los derechos aplicados a todo el Wiki.
 xe.panels.rights.help=Ayuda del editor de privilegios
 xe.panels.rights.users=Usuarios
-xe.panels.rights.usersexplanation=Este campo deber\u00EDa contener el identificador wiki de cada usuario al que desee aplicar los derechos. Por ejemplo, para usuarios no identificados deber\u00EDa usar XWiki.JohnDoe. XWiki.XWikiGuest
+xe.panels.rights.usersexplanation=Este campo deber\u00EDa contener el nombre wiki de cada usuario al que quieras aplicar los derechos. Por ejemplo, <em>XWiki.JuanHerrero</em>.  Para los usuarios no identificados se debe usar <em>XWiki.XWikiGuest</em>.
 xe.panels.rights.groups=Grupos
-xe.panels.rights.groupsexplanation=Este campo deber\u00EDa contener el nombre Wiki de cada grupo a los que desea que se le apliquen los derechos. XWiki.XWikiAllGroup representa el grupo de todos los usuarios con una cuenta en su Wiki que est\u00E1n autenticados.
-xe.panels.rights.groupsvirtualexplanation=xwiki:XWiki.XWikiAllGroup representa al grupo de todos los usuarios aut\u00E9nticados utilizando una cuenta global.
+xe.panels.rights.groupsexplanation=Este campo deber\u00EDa contener el nombre wiki de cada grupo al que quieras aplicar los derechos. <em>XWiki.XWikiAllGroup</em> representa el grupo de todos los usuarios autenticados con una cuenta en tu wiki.
+xe.panels.rights.groupsvirtualexplanation=<em>xwiki:XWiki.XWikiAllGroup</em> representa el grupo de todos los usuarios autenticados que utilizan una cuenta global.
 xe.panels.rights.accesslevels=Niveles de acceso
 xe.panels.rights.accesslevelsexplanation=Este campo debe contener una lista de los niveles de acceso que se quieren aplicar a los usuarios y grupos especificados. Los niveles de acceso disponibles son: admin (administraci\u00F3n), programming (proramaci\u00F3n), register (registro), edit (edici\u00F3n), view (visualizaci\u00F3n), comment (comentarios). Para proteger su Wiki en modo vista y edici\u00F3n utilice "view, edit". Para proteger de a\u00F1adir comentarios utilice "comment".
 xe.panels.rights.allowdeny=Permitir/Denegar
-xe.panels.rights.allowdenyexplanation=Este campo deber\u00EDa contener 'Permitir' para indicar que se dispone del derecho y 'Denegar' para indicar que no se dispone de este derecho. Si se dispone del derecho el Wiki o el Espacio es *exclusivamente* visible o editable para los usuarios o grupos que disponen del derecho.
+xe.panels.rights.allowdenyexplanation=Este campo deber\u00EDa contener <em>Permitir</em> para indicar que se dispone del derecho, y <em>Denegar</em> para indicar que no se dispone de ese derecho. Un derecho <em>permitido</em> significa: "esta wiki, espacio o p\u00E1gina es *solo" visible o editable por los usuarios o grupos especificados".
```

Comment:
Still have to decide about that.

Reply:
This is actually part of the source translation, if we want to change it we have to do it on the source first.

Reply:
`<em>` is actually fine since it's semantic (in HTML5) and not styling. If the translation had put `<i>` or `<b>` it would have been a problem. We'll need to decide what type of HTML is supported in comments. Since XWiki outputs HTML5 ATM (it used to output HTML4 and then XHTML in the past), it means that the HTML in the translations should be consisdered HTML5 since I believe we output directly and don't parse the translations FTM.

Reply:
> it means that the HTML in the translations should be consisdered HTML5 since I believe we output directly and don't parse the translations FTM.

I always XML escape the translations I use. I wouldn't trust the translator to write valid HTML, so I would prefer not to have HTML at all in translations.